migrations/1703253259864-school-default-props.js

```diff
@@ -0,0 +1,76 @@
+const mongoose = require('mongoose');
+// eslint-disable-next-line no-unused-vars
+const { alert, error } = require('../src/logger');
+
+const { connect, close } = require('../src/utils/database');
+
+const { Schema } = mongoose;
+
+const School = mongoose.model(
+	'schools202312111053',
+	new mongoose.Schema(
+		{
+			systems: [{ type: Schema.Types.ObjectId, ref: 'system' }],
+			federalState: { type: Schema.Types.ObjectId, ref: 'federalstate' },
+		},
+		{
+			timestamps: true,
+		}
+	),
+	'schools'
+);
+
+const FederalState = mongoose.model(
+	'federalState202312111053',
+	new mongoose.Schema(
+		{
+			name: { type: String, required: true },
+		},
+		{
+			timestamps: true,
+		}
+	),
+	'federalstates'
+);
+
+module.exports = {
+	up: async function up() {
+		await connect();
+
+		await School.updateMany(
+			{
+				systems: { $exists: false },
+			},
+			{
+				systems: [],
+			}
+		)
+			.lean()
+			.exec();
+
+		const tenant = process.env.SC_THEME;
+		let federalStateName = 'Brandenburg';
+		if (tenant !== 'n21') {
```

Comment:
what about default?

Reply:
yeeh, that would be Brandenburg, set above....This is not cleared with PO, but we have only test schools